### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Keri012** [bitfocus/companion#2058]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Hello there,
I’m having issues since I ‘ve added a 4th streamdeck (an XL one) to my original setup (two 15 keys models and a XL).
It seems that Companion has trouble dealing with the new serial number of this particular XL.
(I don’t think it is a question of power as all the devices are on a powered USB hub)
To be clear, the buttons which refer to a surface change (« set surface with S/N to page ») are no longer working… I have a « ?? » in the surface /controller window instead of having the right serial number. 
It is a prb  as I use a lot this function (I have a lot of « back » buttons on all my pages which allow me to go back to the « home » configuration and a « display set » button allowing me to put all 4 surfaces on the right initial pages.
In fact it seems to be an issue with serial numbers and Companion seems to be lost when the four devices are plugged.
I’ve tried several things like update firmware of all devices, de-install Elgato app (to keep only Companion), re-scan surfaces etc… Tried different versions of Companion etc…
Any ideas ??
Thanks a lot !!

### Environment (please complete the following information)

```markdown
- OS: mac OS 10.15.7
- Browser: Chrome
- Companion Version: 2.2.0
```

**Comment by Julusian** [comment]
As said on facebook, this is a known bug in the UI, that will be fixed in the next release f49930e02518256b8ce1c16fd2532f290c1a53ef

**Comment by Keri012** [comment]
OK,
Thanks so much for all your efforts and the amazing work done 👍🏼
